src/screens/Form.js

```diff
@@ -140,6 +140,17 @@ const FormPage = () => (
           </PropertyValue>
         </Property>
 
+        <Property name="kind">
+          <Description>
+            A string that can be passsed to Form to indicate if a user wants
+            seperate styling for formfield label in which `formField.kind.label`
+            can be used.
+          </Description>
+          <PropertyValue type="string">
+            <Example>{`kind= "string"`}</Example>
```

Comment:
```suggestion
            <Example>"survey"</Example>
```
In the past we haven't included the prop name in the example, just the value that the prop could be

Reply:
aw okay 👍